heisenbridge/channel_room.py

```diff
@@ -22,7 +26,7 @@ def init(self):
 
     @staticmethod
     async def create(network: NetworkRoom, name: str):
-        room_id = await network.serv.create_room('{} ({})'.format(name, network.name), ''.format(name, network.name), [network.user_id])
```

Comment:
This was supposed to be plain `''` due to not having a channel topic at create time.